config/routes.rb

```diff
@@ -3,6 +3,7 @@
   namespace :admin do
     resources :users, only: [:index, :new, :create, :edit, :update, :destroy]
   end
+  resources :expenses
```

Comment:
Would it be a good idea to explicitly define which resource actions you're using, as you have done with `:users`?

Reply:
I think it is ok like that as I'm using all available routes, whereas with users I have the two lots of users routes, one singular, that each only use some of the available routes.

Reply:
👍 
I was thinking that could be your reasoning too.